resources/js/components/SearchForm.vue

```diff
@@ -0,0 +1,161 @@
+<script>
+import Vue from 'vue';
+
+const moment = require('moment');
+require('moment/locale/de');
+import checkView from 'vue-check-view';
+Vue.use(checkView);
+
+export default {
+    name: 'SearchForm',
+    props: {
+        committees: {
+            type: Array,
+            default: () => []
+        }
+    },
+    data() {
+        return {
+            inputValue: '',
+            committeesList: this.committees,
+            sortedCommittees: [],
+            filtered: false
+        };
+    },
+    created () {
+        this.sortByChar();
+    },
+    methods: {
+        sortByChar() {
```

Comment:
is it maybe better to have this as a computed property?
i see that this method is called in `created` hook and here https://github.com/nextsolx/ratsinformationssystem/blob/5a9614c21694b3f94c058c5edd144255a5224bb1/resources/js/components/SearchForm.vue#L53
so i guess there's no need to call it by hand, it could be automatically handled by vue as a computed propery